v2v-helper/migrate/migrate.go

```diff
@@ -262,11 +271,15 @@ func (migobj *Migrate) LiveReplicateDisks(ctx context.Context, vminfo vm.VMInfo)
 		// If its the first copy, copy the entire disk
 		if incrementalCopyCount == 0 {
 			for idx := range vminfo.VMDisks {
+				startTime := time.Now()
+				migobj.logMessage(fmt.Sprintf("Starting full disk copy of disk %d ", idx))
```

Comment:
Please log the timestamp also here

Reply:
The logMessage has timestamps added to the logs, so we skipped adding those explicitly to the messages.